INSTALL.txt

```diff
@@ -168,9 +190,8 @@ MS compilers
 If you are familiar with MS tools, that's obviously the easiest path, and
 the compilers are hopefully more mature (although in my experience, they
 are quite fragile, and often segfault on invalid C code). The main drawback
-is that no fortran compiler + MS compiler combination has been tested -
-mingw-w64 gfortran + MS compiler does not work at all (it is unclear
-whether it ever will).
+is that mingw-w64 gfortran + MSVC does not work at all (it is unclear
```

Comment:
I haven't tried recently, but why would it? Carl's toolchain is all of MinGW, I don't think MSVC + gfortran was even a goal (it would be very nice to have though).